test/models/project_media_7_test.rb

```diff
@@ -196,4 +196,76 @@ def setup
       ProjectMedia.handle_fact_check_for_existing_claim(pm1, pm2)
     end
   end
+
+  test "should save the original_claim url and original_claim_hash when Link Media is created from original_claim" do
+    setup_elasticsearch
```

Comment:
I don't think we need ElasticSearch in this test, right? Please remove if possible (it makes tests slower).